examples/azure-linux-vm-spot-vm/main.tf

```diff
@@ -3,17 +3,11 @@
 # Environment variables can also be used https://www.terraform.io/language/values/variables#environment-variables
 
 module "azurerm_linux_virtual_machine" {
-  source            = "../../"
-  admin_username    = "admin_username"
-  admin_password       = var.admin_password
-  priority            = var.priority
-  max_bid_price       = var.max_bid_price
-  eviction_policy     = var.eviction_policy
-
-  # Set the source_image_reference below based on your naming conventions. The value for source_image_reference provided below is for example illustration purposes only
-  source_image_reference_offer     = "0001-com-ubuntu-server-lunar"
-  source_image_reference_sku       = "23_04-lts-gen2"
-
-
+  source          = "intel/azure-linux-vm/intel"
+  admin_username  = "adminuser"
+  admin_password  = var.admin_password
+  priority        = "Spot"
+  max_bid_price   = 0.0874
+  eviction_policy = "Deallocate"
```

Comment:
Add azurerm_resource_group_name, virtual_network_name, azurerm_network_interface_name, disable_password_authentication. Example errors without these values. Also, please include these as required parameters in the comments and example's readme